### DISCUSSION TITLE
GSOC 2023

### DISCUSSION THREAD

**Discussion by LemarAb** [polypheny/Polypheny-DB#431] [General]
Hey guys, 

Is GSOC 2023 participation planned?

**Comment by vogti** [comment]
Hi @LemarAb 
We do not participate in the Google Summer of Code 2023. However, we plan to participate again next year. There a many remarkable projects participating in the GSoC this year, so I hope you find a cool project for this summer. If you are interested, we would be glad to hear from you again next year :)
